### PR TITLE
#2172 - Remove Part-Time Questions from Application

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Hid the questions for part-time applications;
- Tested Part-time and Full-time application.

## Part-time test:
image
image
image
image
image

## Full-time test:
image
image
image
image
image

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM, nice work @andrepestana-aot

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by ann-aot — APPROVED** [review]
👍 Good work @andrepestana-aot

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 1]
```diff
@@ -10223,9 +10223,9 @@
...
               "properties": {},
               "conditional": {
-                "show": null,
-                "when": null,
-                "eq": "",
+                "show": "true",
+                "when": "howWillYouBeAttendingTheProgram",
+                "eq": "Full Time",
                 "json": ""
               },
```
Comment (dheepak-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json:10230]: I still see this question in PT application. 
image

Is it not the one listed here to be removed ? 
image

**Review by sh16011993 — APPROVED** [review]
Nice Work @andrepestana-aot 👍

**Comment by dheepak-aot** [comment]
Great job. Just one outstanding comment.

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json (hunk 1 above)
Comment (andrepestana-aot) [reply sources/packages/forms/src/form-definitions/sfaa2021-22.json:10230]: Good catch. Done.

**Review by dheepak-aot — APPROVED** [review]
Thanks for the change 👍

**Pull request merged**